x/payments/internal/keeper/subscriptions.go

```diff
@@ -62,7 +63,7 @@ func (k Keeper) EffectSubscriptionPayment(ctx sdk.Context, subscriptionId string
 
 	// Check if should effect
 	if !subscription.ShouldEffect(ctx) {
-		return types.ErrTriedToEffectSubscriptionPaymentWhenShouldnt(types.DefaultCodespace)
+		return sdkerrors.Wrap(types.ErrTriedToEffectSubscriptionPaymentWhenShouldnt, types.DefaultCodespace)
```

Comment:
Should not output DefaultCodespace; can return error directly.

Reply:
ok